wiki/People/The_Team/Global_Moderation_Team/es.md

```diff
@@ -3,46 +3,56 @@ tags:
+*Véase también: [Reporta mala conducta](/wiki/Reporting_bad_behaviour)*
+
+El Equipo de Moderación Global es responsable del bienestar del chat y del foro y se encarga de las interacciones dentro de la comunidad. Deben ser contactados si tienes alguna pregunta pertinente al aspecto de la moderación del juego, que incluye (pero no se limita a):
+
+- problemas con el chat (ej. silencios o acoso)
+- conducta inapropiada en el foro (ej. comentarios inapropiados)
+- la sección de discusión y comentarios de un beatmap
+- moderación del perfil (Contenido NSFW o contenido ofensivo en fotos de perfil, imagen de portada o detalles, ...)
 
 ## Convirtiéndose en un GMT
 
-Convertirse en un miembro del Global Moderation Team requiere de una cantidad notable de contribuciones. Miembros potenciales son comúnmente seleccionados por querer ayudar a otros usuarios, su actividad dentro de la comunidad y tener buen comportamiento. La habilidad de hablar otro idioma aparte del Inglés es sumamente recomendado. Los cantidatos son discutidos por el equipo para determinar su potencial como miembro del equipo.
+Convertirse en un miembro del Equipo de Moderación Global requiere de una cantidad notable de contribuciones. Miembros potenciales son comúnmente seleccionados por querer ayudar a otros usuarios, su actividad dentro de la comunidad y tener buen comportamiento. La habilidad de hablar otro idioma aparte del Inglés es sumamente recomendado. Los candidatos son discutidos por el equipo para determinar su potencial como miembro del equipo.
 
 ### Cómo ser considerado
 
-Los factores más importantes para ser considerado son el comportamiento, la actividad, entendimiento de las reglas y paciencia:
+Los factores más importantes para ser considerado son el comportamiento, la actividad, entendimiento de [las reglas](/wiki/Rules) y paciencia:
 
 1. Demostrar una buena conducta al ser educado y tratar a otros usuarios con respeto. Mantener tu cuenta libre de silencios serios o problemas es esencial.
-2. Demostrar una actividad dentro del juego, participando en el chat y/o en los foros. Reporta cualquier problema o malas conductas vía el sistema de reportes. Responde preguntas y ayuda a otros con sus problemas.
+2. Demostrar una actividad dentro del juego, participando en el chat y/o en los foros. [Reporta malas conductas](/wiki/Reporting_bad_behaviour) o cualquier problema vía el sistema de reportes. Responde preguntas y ayuda a otros con sus problemas.
 3. Tener un entendimiento adecuado de [las reglas](/wiki/Rules) y cómo son aplicadas. Es importante saber qué es aceptable en osu! y qué no lo es.
 4. Ser paciente. El GMT no añade no es espontáneo en cuanto a añadir miembros que lleven ayudando un par de días. Al GMT le toma tiempo evaluar a alguien-su personalidad, con qué pueden ayudar, su actividad, etc.
```

Comment:
Maybe this sentence could be reworked a bit, despite not being part of the PR.

Reply:
Maybe with "4. Ser paciente. Los GMT no añade miembros de manera espontanea que lleven ayudando un par de días. A los GMT les toma tiempo evaluar a alguien su personalidad, con qué pueden ayudar, su actividad, etc."

What u think